### PR TITLE
Bugfix 20240905

### PR DESCRIPTION
**Pull request by itsTheFae**
- [x] I have tested my changes against the `dev` branch (the latest developmental version), and this pull request is targeting that branch as a base
- [x] I have tested my changes on Python 3.8 or higher
- [x] I have ensured my code is formatted using [Black](https://github.com/psf/black)

----

### Description

A few bug fixes for `playnext` & `seek` commands, as well as some tweaks to timers to prevent redundant messages.

### CODE REVIEW

**Comment by BabyBoySnow** [comment]
Did you do any testing on the playnow command to make sure it wasn't having the same behavior? I forgot to get around to it

File: musicbot/bot.py [hunk 1]
```diff
@@ -2637,16 +2637,18 @@ async def handle_vc_inactivity(self, guild: discord.Guild) -> None:
             )
         except asyncio.TimeoutError:
             # could timeout after a disconnect.
-            if guild.me.voice and guild.me.voice.channel:
+            if guild.voice_client and isinstance(
+                guild.voice_client.channel, (discord.VoiceChannel, discord.StageChannel)
```
Comment (BabyBoySnow) [inline musicbot/bot.py:2641]: Did you mean to put whitespace here? Or is mobile just being funny?

Comment (itsTheFae) [reply musicbot/bot.py:2641]: Yes. think it was black formatting.

**Comment by itsTheFae** [comment]
> Did you do any testing on the playnow command to make sure it wasn't having the same behavior? I forgot to get around to it

Didn't run the  playnow command, but the bug was down in `_cmd_play` by not passing head to the underlying entry logic in playlists.  So it would have similarly not queued up correctly.

**Comment by BabyBoySnow** [comment]
Ah ok. Does this also fix play next only working with links?

**Comment by itsTheFae** [comment]
> Ah ok. Does this also fix play next only working with links?

I haven't tested that.  It should work the same as play for most things.  The only thing that changed was passing the head parameter.

**Comment by BabyBoySnow** [comment]
Talk to you more on discord lol

**Pull request merged**